ros2/ament_setup.cpp.tpl

```diff
@@ -0,0 +1,30 @@
+#include "{{header}}"
+
+#include <cstdlib>
+#include <string>
+
+namespace {{namespace}} {
+namespace {
+
+constexpr const char* AMENT_PREFIX_PATH_ENV = "AMENT_PREFIX_PATH";
+constexpr const char* AMENT_PREFIX_PATH = "{{ament_prefix_path}}";
+
+}
+
+void setup_ament_prefix_path(bool append)
```

Comment:
Per the style guide  `SetUpAmentPrefixPath`. I propose `bool SetUpAmentPrefixPath(bool allow_appending)`, handle possible failures of setting up the env vars.